akka-persistence-query/src/main/scala/akka/persistence/query/UpdatedDurableState.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query
+
+import akka.annotation.ApiMayChange
+import akka.annotation.DoNotInherit
+
+/**
+ * API May Change
+ *
+ * The `DurableStateStoreQuery` stream elements for `DurableStateStoreQuery`.
+ *
+ * The implementation can be a [[UpdatedDurableState]] or a `DeletedDurableState`.
+ * `DeletedDurableState` is not implemented yet, see issue https://github.com/akka/akka/issues/30446
+ *
+ * Not for user extension
+ */
+@DoNotInherit
+sealed trait DurableStateChange {
```

Comment:
Did you intend to put this in a file named `UpdatedDurableState.scala`?

Reply:
not intended, thx, I'll change back